File: exercicios/R/1115/1115.r
input = file('stdin','r')

while(TRUE){
    line = strsplit(readLines(input,n=1), " ")
    
    x = as.integer(line[[1]][1])
    y = as.integer(line[[1]][2])

    if(x ==0 || y == 0){
        break
    }
    if(x > 0 && y > 0){
        write("primeiro",'')
    }else if(x < 0 && y > 0){
        write("segundo",'')
    }else if(x < 0 && y < 0){
        write("terceiro",'')
    }else if(x > 0 && y < 0){
        write("quarto",'')
    }
}

File: exercicios/JavaScript/1115/1115.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

while(true){
    var line1 = lines.shift().split(' ');

    var x = parseInt(line1[0]);
    var y = parseInt(line1[1]);

    if(x ==0 || y==0){
        break;
    }
    if(x > 0 && y > 0){
        console.log("primeiro");
    }else if(x < 0 && y > 0){
        console.log("segundo");
    }else if(x < 0 && y < 0){
        console.log("terceiro");
    }else if(x > 0 && y < 0){
        console.log("quarto");
    }
}